src/main/webapp/index.html

```diff
@@ -80,12 +81,28 @@ <h3>Counter</h3>
     margin-right: auto;
     text-align: center;">
 
-        <h1 id="counter"></h1>
-        <form id="counter-form" action="#">
-          <button id="submit" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect">
-            Vote
-          </button>
-        </form>
+      <div id="teachers">
+        <h1>Teachers</h1>
+        <div id="newGame" class="spaced">
+          <a href="teacher/newGameInstance/newGameInstance.html">Click here to start a game!</a>
+        </div>
+        
+        <div id="controlGame" class="spaced">
+          <a href="teacher/controlGameInstance/controlGameInstance.html">Or click here if you want to resume a game you had in progress</a>
+        </div>
+
+        <div id="designGame" class="spaced">
+          <a href="teacher/createGame.html">New here? Click here to design your own game!</a>
+        </div>
+        
+      </div>
```

Comment:
I can't get the CSS rule to space these...

Reply:
You could just add margin to the spaced class, It would look lit his:
![Screenshot 2020-08-06 at 12 13 01 - Display 2](https://user-images.githubusercontent.com/30474787/89561587-5101b780-d7de-11ea-8772-0cf49897427b.png)

Reply:
Awesome, thanks!!